x-pack/test/functional/apps/ml/anomaly_detection/single_metric_job_without_datafeed_start.ts

```diff
@@ -0,0 +1,154 @@
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    it('job creation loads the single metric wizard for the source data', async () => {
+      await ml.testExecution.logTestStep('job creation loads the job management page');
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToJobManagement();
+
+      await ml.testExecution.logTestStep('job creation loads the new job source selection page');
+      await ml.jobManagement.navigateToNewJobSourceSelection();
+
+      await ml.testExecution.logTestStep('job creation loads the job type selection page');
+      await ml.jobSourceSelection.selectSourceForAnomalyDetectionJob('ft_farequote');
+
+      await ml.testExecution.logTestStep('job creation loads the single metric job wizard page');
+      await ml.jobTypeSelection.selectSingleMetricJob();
+    });
+
+    it('job creation navigates through the single metric wizard and sets all needed fields', async () => {
+      await ml.testExecution.logTestStep('job creation displays the time range step');
+      await ml.jobWizardCommon.assertTimeRangeSectionExists();
+
+      await ml.testExecution.logTestStep('job creation sets the time range');
+      await ml.jobWizardCommon.clickUseFullDataButton(
+        'Feb 7, 2016 @ 00:00:00.000',
+        'Feb 11, 2016 @ 23:59:54.000'
+      );
+
+      await ml.testExecution.logTestStep('job creation displays the event rate chart');
+      await ml.jobWizardCommon.assertEventRateChartExists();
+      await ml.jobWizardCommon.assertEventRateChartHasData();
+
+      await ml.testExecution.logTestStep('job creation displays the pick fields step');
+      await ml.jobWizardCommon.advanceToPickFieldsSection();
+
+      await ml.testExecution.logTestStep('job creation selects field and aggregation');
+      await ml.jobWizardCommon.assertAggAndFieldInputExists();
+      await ml.jobWizardCommon.selectAggAndField(aggAndFieldIdentifier, true);
+      await ml.jobWizardCommon.assertAnomalyChartExists('LINE');
+
+      await ml.testExecution.logTestStep('job creation inputs the bucket span');
+      await ml.jobWizardCommon.assertBucketSpanInputExists();
+      await ml.jobWizardCommon.setBucketSpan(bucketSpan);
+
+      await ml.testExecution.logTestStep('job creation displays the job details step');
+      await ml.jobWizardCommon.advanceToJobDetailsSection();
+
+      await ml.testExecution.logTestStep('job creation inputs the job id');
+      await ml.jobWizardCommon.assertJobIdInputExists();
+      await ml.jobWizardCommon.setJobId(jobId);
+
+      await ml.testExecution.logTestStep('job creation displays the validation step');
+      await ml.jobWizardCommon.advanceToValidationSection();
+
+      await ml.testExecution.logTestStep('job creation displays the summary step');
+      await ml.jobWizardCommon.advanceToSummarySection();
+    });
+
+    it('job creation runs the job and displays it correctly in the job list', async () => {
```

Comment:
nit: probably there is no need to start each test suite with `job creation `

Reply:
i agree that it's not needed, but i'm going to keep it in as it is consistent with all of the other job wizard test files in this folder.